config/zones/CA-ON.yaml

```diff
@@ -87,33 +87,26 @@ emissionFactors:
       - datetime: '2021-01-01'
         source: Electricity Maps, 2021 average
         value: 57.47323263502428
+    gas:
+      datetime: '2015-01-01'
+      source: Mallia, E., Lewis, G. "Life cycle greenhouse gas emissions of electricity
+        generation in the province of Ontario, Canada." (2013)
+      value: 497 
     hydro discharge:
-      - datetime: '2015-01-01'
-        source: Electricity Maps, 2015 average
-        value: 53.48322359927377
-      - datetime: '2016-01-01'
-        source: Electricity Maps, 2016 average
-        value: 53.48323094170299
-      - datetime: '2017-01-01'
-        source: Electricity Maps, 2017 average
-        value: 53.03166247658179
-      - datetime: '2018-01-01'
-        source: Electricity Maps, 2018 average
-        value: 48.27082098236083
-      - datetime: '2019-01-01'
-        source: Electricity Maps, 2019 average
-        value: 47.473184887281725
-      - datetime: '2020-01-01'
-        source: Electricity Maps, 2020 average
-        value: 48.54750352993932
-      - datetime: '2021-01-01'
-        source: Electricity Maps, 2021 average
-        value: 57.47323263502428
+      datetime: '2015-01-01'
+      source: Mallia, E., Lewis, G. "Life cycle greenhouse gas emissions of electricity
+        generation in the province of Ontario, Canada." (2013)
+      value: 22
```

Comment:
`hydro discharge` in Electricity Maps refers to discharge from pumped hydro storage, so I think it's inappropriate to use this number here.

(Extra context: The values Electricity Maps has now are yearly averages for IESO carbon intensity, based on logic that pumped hydro is pumped up with electricity from the grid. In any event, the current Ontario parser doesn't report pumped hydro charging or discharging. If I recall correctly, from available data we might be able to get discharge at the Adam Beck pumped reservoir, but I haven't seen charging data.)

Reply:
I see this has been moved to its own hydro key so this seems fine now